queue/memqueue/client.go

```diff
@@ -16,7 +16,7 @@ import (
 type MemQueue struct {
 	Name      string
 	queue     *memqueue.Queue
-	inner     taskq.Factory
+	Inner     taskq.Factory
```

Comment:
Why would you want to make `inner` public? By convention its supposed to be hidden right?

Reply:
It's the cleanest way to access the redis client, we could change the name from `inner`